### DISCUSSION TITLE
Provide immediate feedback when slider ends are missed

### DISCUSSION THREAD

**Discussion by Aldoggen** [ppy/osu#21238] [Ideas]
### Context
In stable, missing a slider end gives a 100 for that slider and displays the 100 at the end of the slider.
In lazer, missing slider ends gives no immediate visual feedback. The only ways to know a slider end has been missed are to notice it while (re)playing or look it up in the end screen.

### Feature Request
It would be nice if lazer had a way to provide immediate feedback upon missing a slider end, like stable does.
This could be implemented as support for a skin element that appears at the position of a slider end when it is missed.
From the user's point of view, the change would either be nonexistent, or purely visual, depending on what skin is used.

### Useful?
I think this is useful, because dropping slider ends has an impact on how good your play is.
It will also make people stop wondering why they suddenly have 99.99% accuracy.

### Skinning Details
I propose a standard for how this skin element would look.
The miss symbol is almost always red, and often spells the word "miss" or is some kind of cross or minus sign.
Therefore, the slider end symbol should be orange. In case the miss symbol is a word, the slider end symbol should be "drop" (from the expression "dropping a slider end"). In case the miss symbol is a symbol, the slider end symbol should be the same symbol.

**Comment by mk56-spn** [comment]
There's already plans to indicate them.

**Reply by Aldoggen** [reply]
Thanks a lot!
Could you point me to the texts that talk about the plans?
